flocx_market/api/app.py

```diff
@@ -16,6 +17,9 @@ def create_app(app_name):
     )
     app.config['PROPAGATE_EXCEPTIONS'] = CONF.flask.PROPAGATE_EXCEPTIONS
     api = Api(app)
-    api.add_resource(Offer, '/offer', '/offers/<string:marketplace_offer_id>')
+    api.add_resource(Offer, '/offer', '/offers/',
+                     '/offers/<string:marketplace_offer_id>')
+    api.add_resource(Bid, '/bid', '/bids/',
+                     '/bids/<string:marketplace_bid_id>')
```

Comment:
The specification requires that we use `/bid` and `/bid/<id>`, not `/bids/<id>`.

Reply:
I know, I talked to @SahilTikale about changing that to follow the REST conventions. Which do you think is the right approach @larsks?

Reply:
I would implement it according to the spec right now, and we can talk with Sahil when he's back if you think we should update the spec.

Reply:
Sounds good